src/transformers/models/glm/modeling_glm.py

```diff
@@ -54,6 +55,9 @@
 from ...modeling_flash_attention_utils import _flash_attention_forward
 
 
+_CHECKPOINT_FOR_DOC = "THUDM/glm-4-9b-chat"
```

Comment:
Maybe use the non-chat version ([THUDM/glm-4-9b](https://huggingface.co/THUDM/glm-4-9b)) for token classification?

Reply:
Seems fair point. Updated as requested. @stevhliu